### PR TITLE
- change link on creator

### PR DESCRIPTION
**Pull request by lennardevertz**

### CODE REVIEW

File: packages/constants/src/links.ts [hunk 1]
```diff
@@ -48,4 +48,4 @@ export const CHROME_EXTENSION_LINK =
   'https://chromewebstore.google.com/detail/idriss/fghhpjoffbgecjikiipbkpdakfmkbmig';
 
 export const PROTOCOL_FEE_USER_GUIDE_LINK =
-  'https://docs.idriss.xyz/user-guides/protocol-fee';
+  'https://docs.idriss.xyz/user-guides/creators';
```
Comment (SolutionsEngineer) [inline packages/constants/src/links.ts:51]: do we want to rename the const to something like `CREATORS_URER_GUIDE_LINK`?

Comment (lennardevertz) [reply packages/constants/src/links.ts:51]: yes

**Review by SolutionsEngineer — DISMISSED** [review]

File: packages/constants/src/links.ts [hunk 2]
```diff
@@ -33,6 +33,8 @@ export const EXTERNAL_ANNOUNCEMENT_LINK = {
 };
 
 export const DOCUMENTATION_LINK = 'https://docs.idriss.xyz/';
+export const EXTENSION_URER_GUIDE_LINK =
```
Comment (SolutionsEngineer) [inline packages/constants/src/links.ts:36]: small type `URER` -> `USER`

Comment (lennardevertz) [reply packages/constants/src/links.ts:36]: ty

**Review by SolutionsEngineer — DISMISSED** [review]

**Review by SolutionsEngineer — DISMISSED** [review]

**Review by SolutionsEngineer — DISMISSED** [review]

**Review by SolutionsEngineer — APPROVED** [review]

**Pull request merged**